Fundamentals/first_letter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016 Ami . All rights reserved.
```

Comment:
The copyright line should be a comment with the following text:
`Copyright (c) 2016 Lightricks. All rights reserved.`